net-dialup/freeradius-client/files/freeradius-client-1.1.7-ar-configure.in.patch

```diff
@@ -0,0 +1,10 @@
+--- a/configure.in	2020-07-08 19:20:38.180447370 +0200
++++ b/configure.in	2020-07-08 19:21:00.840479965 +0200
+@@ -33,6 +33,7 @@
+ 
+ dnl Needed for normal compile
+ AC_PATH_PROG(AR, ar)
++AS_IF([test "${AR}" = ""], [AC_PATH_PROG(AR, x86_64-pc-linux-gnu-ar)])
```

Comment:
You're kind of there. What is this going to think about my aarch64-unknown-linux-gnu-ar?

The idea is to respect AR, whatever it points to. This is usually ${TRIPLE}-ar but it could be anything.

AM_PROG_AR may work?

Reply:
Indeed, it works,

```
checking for x86_64-pc-linux-gnu-ar... (cached) x86_64-pc-linux-gnu-ar
checking the archiver (x86_64-pc-linux-gnu-ar) interface... ar
```

```
libtool: link: x86_64-pc-linux-gnu-ar cru .libs/libfreeradius-client.a  buildreq.o clientid.o env.o sendserver.o avpair.o config.o dict.o ip_util.o log.o util.o rc-md5.o md5.o
```